downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/Main.java

```diff
@@ -0,0 +1,223 @@
+package org.reactome.release.downloadDirectory;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileReader;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.lang.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gk.persistence.MySQLAdaptor;
+
+public class Main {
+	static MySQLAdaptor dbAdaptor = null;
+	private static final Logger logger = LogManager.getLogger();
+	
+	public static void main(String[] args) throws Exception {
+		
+		logger.info("Beginning Download Directory step");
+		String pathToConfig = "src/main/resources/config.properties";
+		Properties props = new Properties();
+		props.load(new FileInputStream(pathToConfig));
+		
+		//TODO: Check stable identifiers db exists
+		//TODO: File existence check and size check
+		//TODO: Parallelize executions
+		//TODO: Integration with Perl wrapper
+		
+		//Set up DB adaptor
+		String username = props.getProperty("username");
+		String password = props.getProperty("password");
+		String database = props.getProperty("database");
+		String host = props.getProperty("host");
+		int port = Integer.valueOf(props.getProperty("port"));
+		String releaseNumber = props.getProperty("release");
+		String releaseDirAbsolute = props.getProperty("absoluteReleaseDirectoryPath");
+		String releaseDownloadDir = props.getProperty("releaseDownloadDirectoryPath");
+		String releaseDownloadDirWithNumber = releaseDownloadDir + releaseNumber;
+		dbAdaptor = new MySQLAdaptor(host, database, username, password, port);
+		File releaseDir = new File(releaseNumber);
+		if (!releaseDir.exists()) 
+		{
+			releaseDir.mkdir();
+		}
+		
+		String pathToSpeciesConfig = props.getProperty("speciesConfigPath");
+		
+		// Determine which steps will be run via stepsToRun.config file
+		String pathToStepConfig = props.getProperty("stepsToRunConfigPath");
+		FileReader fr = new FileReader(pathToStepConfig);
+		BufferedReader br = new BufferedReader(fr);
+		List<String> stepsToRun = new ArrayList<String>();
+		String line;
+		while ((line = br.readLine()) != null) 
+		{
+			if (!line.startsWith("#")) 
+			{
+				stepsToRun.add(line);
+			}
+		}
```

Comment:
This could be done with a stream:

```
List<String> stepsToRun = br.lines().filter(
    line -> !line.startsWith("#")
).collect(Collectors.toList());
```